### ISSUE TITLE
We need macros for statements

### ISSUE THREAD

**Issue by luizffgv** [luizffgv/_Hh_#1]
`if`/`else`, `for`, `while` and friends.

You can actually use them through token pasting but it's extremely tiresome. This language is supposed to be esoteric, not infuriating.

**Comment by elegantShock2258** [comment]
Would recommend sticking to macros for some time, I think we should first finalize some sort of a design before jumping into it.

**Comment by luizffgv** [comment]
Exactly, I meant macros for statements. I do agree that we should have a decent design, like better organization overall and shorter macros for the most common keywords.
